File: repos/Gateway4CSharp/GWDataCenter/MqService/Models/MqEvtMessage.cs
using System;
using System.Collections.Generic;

namespace GWDataCenter.MqService;

public class MqEvtMessage
{
    public string Time { get; set; }
    public string Flow { get; set; }
    public List<EquipEvent> EventItems { get; set; } = [];
}

public class EquipEvent
{
    public int DeviceId { get; set; }
    public List<EquipEventItem> EquipEvents { get; set; } = [];
}

public class EquipEventItem
{
    /// <summary>
    /// 显示到实时快照的内容
    /// </summary>
    public string Msg { get; set; }
    
    /// <summary>
    /// 联动传入的内容，如果为空就传入msg
    /// </summary>
    public string Msg4Linkage { get; set; }
    public MessageLevel Level { get; set; }
    public DateTime OccurDateTime { get; set; }
    public int EquipNo { get; set; }
}

File: repos/Gateway4Java/gwminidatacenter/src/main/java/gwdatacenter/mqservice/MqEvtMessage.java
package gwdatacenter.mqservice;

import gwdatacenter.*;
import java.util.*;
import java.time.*;

public class MqEvtMessage
{
	private String Time;
	public final String getTime()
	{
		return Time;
	}
	public final void setTime(String value)
	{
		Time = value;
	}
	private String Flow;
	public final String getFlow()
	{
		return Flow;
	}
	public final void setFlow(String value)
	{
		Flow = value;
	}
	private ArrayList<EquipEvent> EventItems = new ArrayList<>();
	public final ArrayList<EquipEvent> getEventItems()
	{
		return EventItems;
	}
	public final void setEventItems(ArrayList<EquipEvent> value)
	{
		EventItems = value;
	}
}